localization/english/wc_traits_l_english.yml

```diff
@@ -24,7 +24,7 @@
  trait_creature_naga_desc:0 "The naga little resemble the nobility they once were. Their proud appearance has been warped and distorted, many of them received a number of special changes characteristic of the sea depths they now call their home, such as an ability to release ink, tentacles, sharp scales, poisonous fangs, strong shells of mollusks, serpents instead of hair, large and sharp dorsal fins, and strong tails."
  trait_creature_naga_character_desc:0 "$trait_creature_naga_desc$"
  trait_creature_blood_elf:0 "Blood Elf"
- trait_creature_blood_elf_desc:0 "The blood elves are a race formed from the high elves of Quel'Thalas. In honour of their dead brethren, the survivors call themselves blood elves or Sin'dorei, which in thalassian means "children of blood". Both high elves and blood elves, regardless of class, status or origin, are fond of magic and are somewhat addicted to it. On average, blood elves are slightly taller than humans and considerably thinner in bone structure."
+ trait_creature_blood_elf_desc:0 "The blood elves are a race formed from the high elves of Quel'Thalas. In honor of their dead brethren, the survivors call themselves blood elves or Sin'dorei, which in thalassian means "children of blood". Both high elves and blood elves, regardless of class, status or origin, are fond of magic and are somewhat addicted to it. On average, blood elves are slightly taller than humans and considerably thinner in bone structure."
```

Comment:
```suggestion
 trait_creature_blood_elf_desc:0 "The blood elves are a race formed from the high elves of Quel'Thalas. In honor of their dead brethren, the survivors call themselves blood elves or Sin'dorei, which in Thalassian means "children of blood". Both high elves and blood elves, regardless of class, status or origin, are fond of magic and are somewhat addicted to it. On average, blood elves are slightly taller than humans and considerably thinner in bone structure."
```

Reply:
Oh, good point. For some reason I thought of the adjective here, but you're right, it's the language like English/German etc.